packages/tradershub/src/public/images/ic-derivx-password-updated.svg

```diff
@@ -0,0 +1 @@
+<svg width="122" height="108" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M95.785 100H4.215C1.915 100 0 98.084 0 95.785V4.215C0 1.915 1.916 0 4.215 0h91.57C98.085 0 100 1.916 100 4.215v91.187c0 2.682-1.916 4.598-4.215 4.598z" fill="#BCE4E5"/><path opacity=".5" d="M85.44 82.374H14.56c-2.3 0-3.832-1.914-3.832-3.83V65.515c0-2.299 1.917-3.83 3.832-3.83h70.498c2.298 0 3.831 1.916 3.831 3.83v13.027c.383 1.916-1.533 3.831-3.448 3.831z" fill="#fff"/><path d="M24.904 72.03l2.3 1.15-1.15 1.533-2.3-1.533v2.682H21.84V73.18l-2.3 1.533-1.15-1.533 2.3-1.15-2.3-1.149 1.15-1.915 2.3 1.532v-2.682h1.916v2.682l2.299-1.532 1.15 1.532-2.3 1.533h-.001zm13.41 0l2.3 1.15-1.15 1.533-2.3-1.533v2.682h-1.532V73.18l-2.298 1.533-1.15-1.533 2.299-1.15-2.3-1.149 1.15-1.915 2.299 1.532v-2.682h1.916v2.682l2.299-1.532 1.15 1.532-2.683 1.532zm13.793 0l2.3 1.15-1.15 1.533-2.3-1.533v2.682h-1.915V73.18l-2.3 1.533-1.148-1.533 2.299-1.15-2.3-1.149 1.15-1.532 2.299 1.532v-2.68h1.916v2.681l2.299-1.532 1.15 1.532-2.3 1.15v-.002zm13.41 0l2.3 1.15-1.15 1.533-2.3-1.533v2.682h-1.915V73.18l-2.3 1.533-1.15-1.533 2.3-1.15-2.3-1.149 1.15-1.532 2.3 1.532v-2.68h1.916v2.681l2.299-1.532 1.15 1.532-2.3 1.15v-.002zm13.793 0l2.3 1.15-1.15 1.533-2.3-1.533v2.682h-1.915V73.18l-2.3 1.533-1.148-1.533 2.299-1.15-2.3-1.532 1.15-1.532 2.299 1.532v-2.682h1.916v2.682l2.299-1.532 1.15 1.532-2.3 1.532z" fill="#fff"/><path opacity=".5" d="M50.192 53.257c12.273 0 22.223-9.95 22.223-22.222 0-12.273-9.95-22.223-22.223-22.223S27.97 18.762 27.97 31.035c0 12.272 9.949 22.222 22.22 22.222z" fill="#fff"/><path d="M58.62 29.503H40.996c-1.15 0-2.299 1.15-2.299 2.299v10.727c0 1.15 1.15 2.3 2.3 2.3H58.62c1.15 0 2.3-1.15 2.3-2.3V32.184c.382-1.531-.767-2.681-2.3-2.681zM51.724 41.38H48.66l.766-4.215c-.766-.383-1.15-1.15-1.15-1.916 0-1.15.767-2.299 2.3-2.299 1.15 0 2.298.767 2.298 2.3 0 .767-.383 1.532-1.15 1.916l.001 4.215zm-6.897-13.41l-.383-1.532c-.766-3.066 1.15-5.747 4.215-6.514 3.065-.767 5.747 1.15 6.513 4.215l.383 1.533 3.065-.768-.383-1.532c-.766-4.598-5.364-7.662-9.96-6.514-4.599.767-7.664 5.364-6.515 9.962l.383 1.534 2.682-.383z" fill="#72AEB2"/><path opacity=".2" d="M62.452 89.655H37.548c-.766 0-1.15-.383-1.15-1.15 0-.765.384-1.149 1.15-1.149h24.904c.766 0 1.15.384 1.15 1.15 0 .766-.384 1.15-1.15 1.15z" fill="#000"/><path d="M100 108c12.15 0 22-9.85 22-22s-9.85-22-22-22-22 9.85-22 22 9.85 22 22 22z" fill="#72AEB2"/><path d="M107.383 94.752h-4.224l-3.624-6.168h-.072l-3.528 6.168H92l5.376-8.664L92.263 78h4.248l3.264 5.664h.072L103.16 78h3.936l-5.16 8.16 5.447 8.592z" fill="#fff"/></svg>
```

Comment:
Could we use Quill-icons?

Reply:
Let me check

Reply:
I did not find these 3 icons in the quill icons

Reply:
Do you need to ask from the Quill-icon team to provide them to us?

Reply:
We will create a list of the icons which we need and will give it to the quill-icon team